src/utils/fastmath.h

```diff
@@ -78,8 +78,11 @@ inline float FastLog(const float a) {
 inline float FastExp(const float a) { return FastPow2(1.442695040f * a); }
 
 // Fast logit for more readable code.
+// This is actually 0.5 * logit((a+1)/2) = atanh(a).
 inline float FastLogit(const float a) {
-  return 0.5f * FastLog((1.0f + a) / (1.0f - a));
+  // Scale Q slightly to avoid logit(1) = infinity.
+  float b = 0.9999999f * a;
```

Comment:
This is not the place for the scaling.

Reply:
So you suggest it would be better to introduce another function 
```inline float ScaledFastLogit(const float a) {
  return FastLogit(0.9999999f * a);
}```
?

Reply:
No, just do it like it was before. If you dislike repeating the numeric value, you can define a constant.

Reply:
What's the best way to get a single constant to both `search.cc` and `node.h`? I haven't figured out how to pass it into the latter.

Reply:
I decided to try passing it in through the function.